index.js

```diff
@@ -0,0 +1,59 @@
+const express = require('express')
+const exphbs = require('express-handlebars')
+const MongoClient = require('mongodb').MongoClient
+const ObjectId = require('mongodb').ObjectId
+const cors = require('cors')
+const dateRange = require('./dateRange')
+
+const url = "mongodb://root:password@localhost:27017"
+
+const app = express()
+app.use(express.json())
+
+const port = 3000
+
+app.engine('handlebars', exphbs());
+app.set('view engine', 'handlebars');
+
+app.use(express.static('public'))
+
+app.use(cors())
+app.options('*', cors())
+
+app.get('/hotels', async (request, response) => {
```

Comment:
both your route callbacks should be split into controller functions in a separate file then imported here, keep it tidier